webapp/src/components/preview-movies/fetch-movies/MoviesByGenre.tsx

```diff
@@ -1,17 +1,17 @@
 import React, { useEffect, useState } from 'react';
 import { getGlobalInstance } from 'plume-ts-di';
 import useLoader from '../../../lib/plume-http-react-hook-loader/promiseLoaderHook';
-import PreviewMoviesService from '../../../services/preview-movies/PreviewMoviesService';
 import { Production } from '../../../api/types/MovieDbTypes';
 import Row from '../style/row/Row';
+import MovieService from '../../../services/movie/MovieService';
 
 function MoviesByGenre({ genre, id_genre }:{ genre:string, id_genre: number }) {
```

Comment:
crée un type pour tes props:

```suggestion
type Props = {
  genre:string,
  idGenre: number
}

function MoviesByGenre({ genre, id_genre }:Props) {
```